app.js

```diff
@@ -2,13 +2,14 @@
 //---------------------------------------------------------------------
```

Comment:
I just have to say that your commenting is really really great.  I am really not very good at commenting my code, but seeing your comments makes me want to add more comments.  If you ever decide to contribute to open source projects as your career progresses, people will love this.